interfaces/builtin/pluggable_storage.go

```diff
@@ -0,0 +1,71 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const pluggableStorageConnectedPlugAppArmor = `
+# Description: Can mount and unmount removable storage. This is restricted
+# because it gives privileged access to mount commands and should only be used
+# with trusted apps.
+# Usage: reserved
+
+# Needed for mount/unmount operations
+capability sys_admin,
+
+# Mount/unmount USB storage devices
+mount /dev/sd* -> /var/snap/${SNAP_NAME}/${SNAP_REVISION}/**,
+umount /var/snap/${SNAP_NAME}/${SNAP_REVISION}/**,
+
+# Mount/unmount SD cards
+mount /dev/mmcblk* -> /var/snap/${SNAP_NAME}/${SNAP_REVISION}/**,
+umount /var/snap/${SNAP_NAME}/${SNAP_REVISION}/**,
+
+# Allow calling the system mount/umount binaries to do the dirty work
+/bin/mount ixr,
+/bin/umount ixr,
+
+# mount/umount (via libmount) track some mount info in these files
+/run/mount/utab wrl,
```

Comment:
This is an info leak similar to mount-observe: mount arguments might contain sensitive information and may be able to enumerate what snaps are installed.